.github/workflows/publish-to-dockerhub.yml

```diff
@@ -0,0 +1,79 @@
+
+name: Publish Docker image
+
+on:
+  push:
+    branches: build-docker-images
+  release:
+    types: [published]
+  # Allow workflow to be triggered manually.
+  workflow_dispatch:
+
+jobs:
+  push_sbtc_to_registry:
+    name: Push sBTC Docker Image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          # We use ubuntu-latest which implies amd64
+          context: "{{ defaultContext }}"
+          file: ./devenv/sbtc/docker/Dockerfile
+          # TODO: Set up docker publishing to use the tag name when publishing and to do
+          # something else when triggered manually. $\{{ github.event.release.tag_name }}
+          tags: blockstack/sbtc:sbtc
+          push: true
+          platform: amd64
+
+  push_services_to_registry:
+    strategy:
+      matrix:
+        service: [
+          electrs,
+          bitcoin,
+          miner,
+          sbtc-bridge-api,
+          sbtc-bridge-web,
+          stacks,
+          stacks-api,
+          stacks-explorer,
+        ]
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        if: ${{ matrix.service }} == sbtc
```

Comment:
Is sbtc not in the matrix?

Reply:
sBTC is above, because it's built with the sources here I put it separately.

Reply:
same question as @CAGS295  - is there a reason `sbtc` can't be added to this matrix that i'm not seeing here?
the build step looks compatible.

Reply:
`if: ${{ matrix.service }} != sbtc` No?

Reply:
I understand that the check means to only run if it is not sbtc. Otherwise, I don't understand the purpose of the if condition.